CoffeeMachine.java

```diff
@@ -0,0 +1,90 @@
+package Lesson_12;
+
+import java.util.Scanner;
+
+public class CoffeeMachine {
+    private String esspresso = "Esspresso";
+    private String macchiato = "Macchiato";
+    private String cappuccino = "Cappuccino";
+    private String milk = "Milk";
+    private String hotChocolate = "Hot Chocolate";
+    private String drinkChoice;
+    private int answer;
+    private char choice;
+    private String[] drinkList = {"Espresso", "Macchiato", "Cappuccino"};
+
+    public CoffeeMachine(){
+        this.esspresso = esspresso;
+        this.macchiato = macchiato;
+        this.cappuccino = cappuccino;
+        this.milk = milk;
+        this.hotChocolate = hotChocolate;
+        this.drinkChoice = drinkChoice;
+        this.answer = answer;
+        this.drinkList=drinkList;
+        this.choice=choice;
+    }
+
+    public void getDrink(){
+        System.out.println("Choose your drink");
+        int n = 0;
+        for (String menu:this.drinkList){
```

Comment:
When you need index, you can simply use plain old for loop:
```
for (int i = 0 ; i < this.drinkList ; i++) {

}
```
It is better approach than using outside counter (in your case variable n)